lsan_suppressions.txt

```diff
@@ -0,0 +1,3 @@
+# Some leaks are detected from libfuse, they don't affect any real code
+# Fix is pending in https://github.com/cberner/fuser/pull/390
+leak:*libfuse.so*
```

Comment:
Please don't pollute the repo root dir, put this inline into the workflow if possible, or at least put it in a relevant subdir (maybe `lact-daemon/src/tests`)